Makefile.overrides.mk

```diff
@@ -1,2 +1,17 @@
 # Use the build container by default
 BUILD_WITH_CONTAINER ?= 1
+
+# Cannot run root tests in container unless the container is privileged,
+# so if you set both those flags we will attempt to run the build container as privileged.
+ifeq ($(TEST_MODE), root)
```

Comment:
This should be the default. If you don't want to/can't do this, we should have something like Go's testing.short()

Reply:
The problem with making root tests default-enabled is that since we also default to container builds, we are by-default asking people to run privileged containers that could very well trash their box in a non-recoverable fashion.

That's not a great default, and in general one of the reasons why I don't like unit tests that require root.

I don't want to make root runs the default unit test target - if we are concerned that this means test coverage is skipped, we should rewrite those tests with mocks to stub out OS functionality, or move them to integration tests (which is arguably where most of these belong anyway)